linux-headers/linux/kvm.h

```diff
@@ -1415,6 +1416,7 @@ struct kvmft_set_master_slave_sockets {
 };
 #define KVMFT_SET_MASTER_SLAVE_SOCKETS    _IOW(KVMIO, 0xcf, struct kvmft_set_master_slave_sockets)
 
+#define KVM_SHM_CANCEL_TIMER       _IO(KVMIO, 0xd2)
```

Comment:
It same as #define KVM_RESET_PUT_OFF _IOW(KVMIO, 0xd2, int)
please change another numver

Reply:
I have modified the magic number to 0xdd, and there's no conflict.